runtime/sema/check_member_expression.go

```diff
@@ -345,16 +345,21 @@ func (checker *Checker) visitMember(expression *ast.MemberExpression, isAssignme
 	//
 	// This would result in a bound method for a resource, which is invalid.
 
-	if !checker.inInvocation &&
```

Comment:
Can `inInvocation` be removed now?

Reply:
I tried removing that, but there were some tests failing, and seemed there is still one use-case where it is required to know whether the current expression is inside an invocation, but not necessarily the immediate child. We could maybe revisit that, but would probably need some additional work.

Reply:
Sounds good!